src/repl.js

```diff
@@ -96,21 +97,34 @@ Prototype REPL - https://github.com/nodejs/repl`,
     const evaluateResult = await this.eval(line, awaited);
 
     if (evaluateResult.exceptionDetails) {
+      // lets try for recovering
+      const result = isRecoverableError(evaluateResult.exceptionDetails.exception, line);
+      if (result) {
+        this.io.setPrefix('...');
+        return $multiLine;
+      } else {
+        // we tried our best - throw error
+        // this.io.setMultiline(false);
+        this.io.setPrefix('>');
```

Comment:
arrow should be reset by io when multiline ends. also the arrow needs a space after it.